worker/edge/edge_test.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+)
+
+const testURL = "http://localhost/vod/test.mp4/playlist.m3u8"
+
+func BenchmarkValidateToken(b *testing.B) {
+	str, err := prepareJWT(time.Hour, testURL)
+	if err != nil {
+		b.Fatal(err)
+	}
+	r, _ := http.NewRequest("GET", testURL+"?jwt="+str, nil)
```

Comment:
```suggestion
	r, _ := http.NewRequest(http.MethodGet, testURL+"?jwt="+str, nil)
```
Alternatively, `http` constants could be used here. But doesn't really matter.